src/state/apis/zapper/zapperApi.ts

```diff
@@ -164,4 +196,33 @@ export const zapperApi = createApi({
   }),
 })
 
+// https://docs.zapper.xyz/docs/apis/getting-started
+export const zapper = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'zapper',
+  endpoints: build => ({
+    getZapperUniV2PoolAssetIds: build.query<GetZapperUniV2PoolAssetIdsOutput, void>({
+      queryFn: async (_, { dispatch }) => {
+        const maybeZapperV2AppTokensData = await dispatch(
+          zapperApi.endpoints.getZapperAppBalancesOutput.initiate(),
+        )
+
+        if (!maybeZapperV2AppTokensData.data) throw new Error('No Zapper data, sadpepe.jpg')
+
+        const zapperV2AppTokensData = Object.values(maybeZapperV2AppTokensData.data)
+
+        const data = zapperV2AppTokensData.map(appTokenData =>
+          toAssetId({
+            chainId: zapperNetworkToChainId(appTokenData.network)!,
+            assetNamespace: 'erc20', // TODO: bep20
```

Comment:
should be fairly straightforward to tackle this in this PR - this is a bug waiting to happen

Reply:
Discussed this one out, we're only fetching data for Ethereum so not a concern for now